nutridine/src/screens/restaurants/RestaurantsPage.js

```diff
@@ -91,36 +91,31 @@ function RestaurantsPage() {
   }
 
   return (
-    <VStack spacing={5} align="stretch">
-      <Flex justify="center" w="full" px={{ base: 4, md: 0 }}>
-        <Input
-          placeholder="Search restaurants..."
-          value={searchTerm}
-          onChange={(e) => setSearchTerm(e.target.value)}
-          bg={inputBg}
-          p={4}
-          size="lg"
-          rounded="md"
-          shadow="base"
-          w={inputWidth}
-          height="56px"
-          fontSize="lg"
-          focusBorderColor="blue.500"
-          sx={{
-            "::placeholder": {
-              fontSize: "lg",
-            },
-          }}
-        />
-      </Flex>
+    <Box
+      position="relative"
+      px={[8, 8, 0]}
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+    >
+      <Input
+        placeholder="Search for restaurants..."
+        width={["85vw", "70vw", "550px"]}
+        mb={4}
+        height="50px"
+        borderRadius="15px"
+        value={searchTerm}
+        onChange={(e) => setSearchTerm(e.target.value)}
+      />
+
       <SimpleGrid
         columns={{ base: 1, md: 2, lg: 3 }}
-        spacing={5}
-        px={{ md: 4 }}
+        spacing={7}
+        px='0'
```

Comment:
Also, for stylings like this, we need to use the predefined spacings eg.md here and not 15px, to have it consistent throughout the app and use tokens instead of primitive variables which is unconventional

Reply:
This doesn't matter right now